manual/settings.md

```diff
@@ -1429,6 +1430,19 @@ Prompting from passwords require interactive shells; if the standard input is no
 
 Default: **true**.
 
+<a name="s3"></a>
+## S3 Settings
+
+Settings applicable for reading from AWS S3 URLs.
+
+#### --s3.clientCacheSize<br />--dsbulk.s3.clientCacheSize _&lt;number&gt;_
+
+The size (count) of the S3Client cache. Since each S3 URL
```

Comment:
I pretty much just picked an arbitrary number. :man_shrugging: In our use case, we're using the same credentials for all the URLs, so the cache only ever has one entry. I suspect using the same credentials for all the URLs will be the usual use case, and it would be a bit of a pain (and somewhat unexpected) to supply more than 20 different sets of credentials in a single `urlfile`. I suppose that is actually an argument for a cache size of 1. :upside_down_face: 

I'm happy to add a sentence explaining this if you want, though. Do note, however, that this file (_settings.md_) and _application.template.conf_ are automatically generated from _dsbulk-reference.conf_, so the same explanatory statement will be shared between the three files.